### ISSUE TITLE
Commands in command palette often repeated when read by screenreader on linux

### ISSUE THREAD

**Issue by webczat** [microsoft/vscode#108857]
I am using the latest vscode insiders but this bug appears for a long time, at least on linux with orca screenreader.
Generally if you open command palette or something similar like symbol list, for example by pressing f1, and start arrowing through the list, sometimes you will be hearing more than one option at a time. For example:
After first press of down arrow I hear "calls: show incoming calls" then "calls: show call hierarchy calls: show incoming calls" etc. I am never fully sure what option I am actually on when that happens.

**Comment by chrmarti** [comment]
Cannot reproduce with VoiceOver.

**Comment by webczat** [comment]
that just confirms it is likely linux specific, as I've thought :) thanks

**Comment by jvesouza** [comment]
This problem happens in my environment and is quite annoying.

**Comment by jvesouza** [comment]
It also happens in my environment when I use the command go to symbol in editor, (ctrl+shift+o.

**Comment by webczat** [comment]
it seems to happen everywhere, just not sure what component is used in things like command palette, go to symbol, remote extension's file selection and whatever else.

**Comment by isidorn** [comment]
@jvesouza @webczat is this something new? Was this happening before? Since I believe we did not do any changes in this area. Can you reproduce on our web playground (so we check if latest Chrome fixes it) https://vscode-web-test-playground.azurewebsites.net/?enter=true&cs=microsoft/vscode

fyi @joanmarie

**Comment by webczat** [comment]
okay. that bug happens for a quite long time, but not sure how long, as there were times when it didn't happen. I cannot reproduce on the playground. It is then likely that it's indeed a chrome bug that was fixed, or something else...

**Comment by jvesouza** [comment]
The bug has been around for a long time, although I don't know exactly when it first appeared. I think I mentioned it in some other bug but honestly I couldn't find what bug I commented on.

Like @webczat I also cannot reproduce on the playground.
